SQL.js

```diff
@@ -138,6 +146,8 @@ class SqlStatement {
 
     return this
   }
+
+  [SQLSymbol] = true
```

Comment:
not sure if we can rely on `instanceof`, so have used the same technique as `unsafe`

Reply:
I think you can and I believe it would lead to a clearer implementation. using a symbol is a more generic approach but I don't think it's required in this case.